setup/so-whiptail

```diff
@@ -28,6 +28,14 @@ whiptail_airgap() {
 	whiptail_check_exitstatus $exitstatus
 }
 
+whiptail_avoid_default_hostname() {
+	[ -n "$TESTING" ] && return
+
+	whiptail --title "Security Onion Setup" \
+		--yesno "We suggest avoiding the default hostname of 'securityonion' in a distributed environment." 8 75 \
```

Comment:
Avoid personalizing messages like "We suggest...". Also, the "Use" option is unclear. Consider: "To prevent hostname conflicts, avoid using the default 'securityonion' hostname in a distributed environment. You can choose to use this default hostname anyway, or change it to a new hostname."

Reply:
Menu uses suggested text now. Also changed `Use` button to `Use Anyway` to mirror wording in description + for clarity.